### PR TITLE
Fixed File layout for Franz 5

### PR DESCRIPTION
**Pull request by agodlydeciple**
updated package.json and webview.js to work with the newest version of Franz.

### CODE REVIEW

**Comment by agodlydeciple** [comment]
after Merge, feature request needs to be submitted to Franz repo Feature section in order to be added.

File: package.json [hunk 1]
```diff
@@ -1,23 +1,14 @@
...
-    "hostedOnly": false,
-    "webviewOptions": {
-      "disablewebsecurity": ""
-    },
-    "openDevTools": false
-  }
-}
+    "id": "xmpp",
+    "name": "xmpp",
+    "version": "1.0.0",
```
Comment (alexander-schranz) [inline package.json]: can you increase the version 2.0.0?

**Review by alexander-schranz — CHANGES_REQUESTED** [review]

**Comment by stevenroose** [comment]
Does this mean that this is currently broken with latest Franz?

**Review by alexander-schranz — APPROVED** [review]

**Comment by alexander-schranz** [comment]
@stevenroose this PR should fix the Franz 5 support

**Comment by alexander-schranz** [comment]
@agodlydeciple thank you

**Pull request merged**